mne/viz/evoked.py

```diff
@@ -126,6 +127,35 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos=None, colors=None, axis=None, outlines='head'):
+    """Helper function to plot color/channel legends for butterfly plots
+    with spatial colors"""
+    from mpl_toolkits.axes_grid.inset_locator import inset_axes
+    ax = inset_axes(axis, width=0.5, height=0.5, loc=3)
+
+    pos, outlines = _check_outlines(pos, outlines, None)
+    pos_x = pos[:, 0]
+    pos_y = pos[:, 1]
+
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_frame_on(False)
+    if any([not pos_y.any(), not pos_x.any()]):
+        raise RuntimeError('No position information found, cannot compute '
+                           'geometries for topomap.')
+
+    for x, y, c in zip(pos_x, pos_y, colors):
+        ax.add_artist(patches.Circle(xy=(x, y), radius=0.03, color=c))
```

Comment:
My bad ... accidentally deleted the import. I don't understand the AttributeError. What context does it appear in?